### ISSUE TITLE
Oops, something went wrong.  Please report this bug with the details below. Report on GitHub

### ISSUE THREAD

**Issue by School-Kris** [PowerShell/PSReadLine#2038]
Keep getting this, and have no idea why

Last 200 Keys:
 b i n \ j a v a . e x e ' Space ' - D f i l e . e n c o d i n g = U T F - 8 ' Space ' - c p ' Space ' C : \ U s e r s \ c b u d d \ A p p D a t a \ R o a m i n g \ C o d e \ U s e r \ w o r k s p a c e S t o r a g e \ b c 5 9 f e 8 b a 6 e f 0 6 5 a e 2 3 5 e e f e d 2 a 5 9 7 1 4 \ r e d h a t . j a v a \ j d t _ w s \ M u s i c P l a y l i s t M a n a g e r V 1 _ a 5 b a c f a b \ b i n ' Space ' M a i n ' Space Enter

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)

**Comment by astef** [comment]
Happened with me too. After quickly switching tab in VS code to "Terminal" and right-clicking, to paste multi-line string. Not reproduces anymore:

```
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 1 Keys:

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -2.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.Paste(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

```
PS> Get-Host | Select-Object Version

Version
-------
5.1.18362.1171
```

**Comment by larbisahli** [comment]
n

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.

**Comment by himansh4333H** [comment]
How to fix this please explain briefly
